shopfloor_packing_info/services/checkout.py

```diff
@@ -10,7 +10,7 @@ class CheckoutExt(Checkout):
 
     def _data_for_packing_info(self, picking):
         res = super()._data_for_packing_info(picking)
-        if picking.picking_type_id.shopfloor_display_packing_info:
+        if picking.note:
```

Comment:
@jbaudoux I don't see how this is supposed to work:
shouldn't we keep the current version with `shopfloor_packing_info` in the `select_package` screen, and use `picking.note` in the screen where we select a picking?